libvast/src/system/node.cpp

```diff
@@ -283,6 +282,13 @@ spawn_accountant(node_actor::stateful_pointer<node_state> self) {
 } // namespace
 
 caf::message status_command(const invocation& inv, caf::actor_system&) {
+  if (caf::get_or(inv.options, "vast.node", false)) {
```

Comment:
The reason `status` for spawned nodes has been disabled is that the new [catalog loading handler call in the index](https://github.com/tenzir/vast/pull/2771/files#diff-d59f67c3b5fcc2153473271c6457c878430b53982892f4988d4116e6c19a0f8fR1266) would cause a race condition -> timeout in spawned nodes - and also I've been told that spawned nodes being able to launch `status` would not make sense semantically

Reply:
I don't understand what race to be honest. The status output of the catalog may be empty because the partitions haven't been merged yet, but what does it have to do with loading the taxonomies?

Reply:
1. The index starts up, loads partitions
2. Query arrives at the index, is deferred until partitions are available
3. Index is done loading partitions, sends them to catalog
4. Catalog loads taxonomies
5. Query is at the catalog, resolves taxonomies
6. Resolves query and catalog lookup result go back to index

The race is between (4) and (5), which can happen in any order. This was an issue with the type-registry as well, but that just happened to load a lot faster.

Once we remove the type-registry state we shouldn't have this race anymore.

Regardless, `vast -N status` doesn't make much sense to support, which is why I suggested disabling it so we can move forward faster with this PR.

Reply:
`vast -N status` will remain forbidden.